src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -857,6 +862,34 @@ export const ComputeModalBase = ({
     )
   }
 
+  const renderCustomTimeoutInMinutes = () => {
+    return h(IdContainer, [
+      id => h(div, {}, [
+        div({ style: { margin: '0.5rem 0' } }, [
+          label({ htmlFor: id, style: { ...computeStyles.label } }, ['Creation Timeout Limit']),
+          h(InfoBox, { style: { marginLeft: '0.5rem' } }, [
+            'Custom and Legacy image creation may take longer than the default 10 minute timeout window. ' +
+            'To avoid an error, you may enter a value between 10 and 45 minutes.'
```

Comment:
FYI, I think that @Qi77Qi mentioned that it would time out after 30 minutes anyway, so maybe replace 45 by 30 here?
I put a maximum of 30 minutes in the swagger doc so we should be consistent.

Reply:
yea, taht's correct..we could change it to higher if needed in leo side